.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text eol=crlf
```

Comment:
Yes I agree for the second one, but it overrides the first one, which is then useless... I'm not sure why you keep both in this file.